ci-operator/step-registry/telco-bastion/setup/telco-bastion-setup-commands.sh

```diff
@@ -4,19 +4,17 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-echo "************ telco-bastion setup command ************"
-
+echo "************ telco cluater setup command ************"
 # TODO: Remove once OpenShift CI will be upgraded to 4.2 (see https://access.redhat.com/articles/4859371)
 ~/fix_uid.sh
 
 # Workaround 777 perms on secret ssh password file
-SSH_PASS=$(cat /var/run/ssh-pass/password)
+KNI_SSH_PASS=$(cat /var/run/kni-pass/knipass)
 
 cat << EOF > ~/inventory
 [all]
-sshd.bastion-telco ansible_ssh_user=tester ansible_ssh_common_args="-o ConnectTimeout=5 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ServerAliveInterval=90" ansible_password=$SSH_PASS
+10.19.16.50 ansible_ssh_user=kni ansible_ssh_common_args="-o ConnectTimeout=5 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ServerAliveInterval=90" ansible_password=$KNI_SSH_PASS
```

Comment:
should we keep the hypervisor ip in a variable?

Reply:
I don't think it's required, though it's a good question. We can create a secret of it I think, but because it's only internal net I doubt if it's needed ?